packages/tools/examples/changeColormap/index.ts

```diff
@@ -0,0 +1,247 @@
+import {
```

Comment:
Thanks for adding this demo, very nice. You need to also add the description to the example-info.json for it to appear